src/controllers/userController.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
to be consistent with other parts of the project, name files like: `user-controller.js` (kebab-case)